### ISSUE TITLE
Scatter Plots

### ISSUE THREAD

**Issue by HugoDF** [yaronn/blessed-contrib#100]
Would having a scatter plot capability be something blessed-contrib is looking to add?

It would make it a very good option to quickly visualise data without having to go through a web interface (where of course we can use D3). This is something I feel where the JavaScript ecosystem is lacking when compared to say Python (eg. matplotlib).

**Comment by yaronn** [comment]
Hi @HugoDF , I think a scatter plot would be a great addition to blessed-contrib (and the js eco system)! I probably don't have capacity to implement it right now but would me more than happy to accept a PR :)

**Comment by HugoDF** [comment]
Thanks for the swift reply @yaronn.
Any pointers on how you would go about implementing it? From the other chart type implementations I'm guessing some canvas plotting would do the trick?

**Comment by yaronn** [comment]
yes, the line chart (https://github.com/yaronn/blessed-contrib/blob/master/lib/widget/charts/line.js) or donut (https://github.com/yaronn/blessed-contrib/blob/master/lib/widget/donut.js) are examples of how to use the canvas. Generally speaking you would implement setData() to paint on a canvas similarly to how you would do it in javascript on the browser.

**Comment by syzer** [comment]
👍

**Comment by vaderdan** [comment]
👍

**Comment by cauboy** [comment]
👍

**Comment by 1inuxoid** [comment]
👍

**Comment by idris-maps** [comment]
:+1:

**Comment by hexagon6** [comment]
:+1:

**Comment by eamonpenland** [comment]
is anyone looking for this still?
